content/guides/code-splitting-import.md

```diff
@@ -26,6 +26,7 @@ function determineDate() {
 
 determineDate();
 ```
+T> Keep in mind that `import([path])` path cannot be fully dynamic. Rather either completely static or partially static.
```

Comment:
Please give examples for 
* fully dynamic: ?
* partially static: `import('./locale/' + language + '.json')`
* completely static: `import('./locale/de.json')`